src/events/2025-01-15-workshop.md

```diff
@@ -2,35 +2,32 @@
 title: "Workshop #1 - Project kick-off workshop: January 15, 2025"
 date: January 15, 2025
 time: 2 - 4 pm Eastern Time
-location: Virtual, via zoom. A link will be shared with the participants, along with a calendar invitation
+location: Virtual, via zoom
 topic: Zebrafish Toxicology Phenotype Atlas project kick-off workshop
-signup: https://forms.gle/Ciqz2ofhwjo7SApM6
+number of paticipants: 46
 ---
 
-* We will introduce the project, discuss goals, and plan the community's involvement (including communication).
+**Workshop goals and content**
+* We introduced the project, reviewed goals, and outlined how the community will engage, including communication plans.
 
-* We will review SOPs and protocols currently used, as well as images of the most common/strangest/hardest to annotate phenotypes encountered.  This information will inform the discussions.
+* We asked participants to share the following information if available:
```

Comment:
This is weird out of context. It would make more sense to say "we asked participants to share 9(...). We used these images in an exercise (...)